### ISSUE TITLE
/mnt/wslg missing

### ISSUE THREAD

**Issue by pixelplant** [microsoft/wslg#893]
### Windows build number:

22000.1219

### Your Distribution version:

20.04

### Your WSL versions:

WSL version: 1.0.0.0
Kernel version: 5.15.74.2
WSLg version: 1.0.47
MSRDC version: 1.2.3575
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22000.1219

### Steps to reproduce:

1. I am not 100% sure if this caused it, but the only different thing that I did was install the vGPU Intel and Nvidia drivers.

### Expected behavior:

expecting WSLg to work and have the folder /mnt/wslg present in the distribution

### Actual behavior:

/mnt/wslg does not exist

I have spent an entire day going through all the tickets on the issue but for the life of me cannot figure out what is wrong and why it seems I am the only one that does not even have the /mnt/wslg folder present. /mnt/wsl is there.

**echo $DISPLAY** returns an empty line
**glxinfo -B** returns "Error: unable to open display :0"

I have an integrated Intel video card and a dedicated Nvidia Geforce GTX 1650 Ti so I went and installed both drivers from the vGPU page ([from here:](https://github.com/microsoft/wslg#pre-requisites))

I even did a complete uninstall of wsl and the virtualized platform followed by a "wsl --install" and I saw that in the installation wsl also installed the GUI apps support so wslG should be there, but it's not the case.

Anyone else experiencing this issue? Since /mnt/wslg does not exist there are no logs that I can send

**Comment by elsaco** [comment]
@pixelplant is `/mnt/wslg` missing or anything that should be under that path? And do you have any distros installed?  What is the output of `wsl -l -v`?

When a WSL distro is run it is mounted under `/mnt/wslg/distro`.

I'm asking about `/mnt/wslg` because that path is hardcoded. Please take a look at `WSLGd/commmon.h`:

```
#define SHARE_PATH "/mnt/wslg"
#define USER_DISTRO_MOUNT_PATH SHARE_PATH "/distro"

```
You could try to open a shell to the `system` distro with `wsl --system` and see if you get more feedback. Also, if you add `debugConsole = true` to your `.wslconfig` it will open a console anytime you start a WSL distro.

**Comment by pixelplant** [comment]
@elsaco /mnt/wslg is missing completely. I only have /mnt/c,d and wsl, no wslg.

I already had an ubuntu 20.04 distro installed and it does not show up. I already updated the distro.
I also installed a fresh ubuntu 22.04 distro and I have the same issue, /mnt/wslg is also missing.

In the end I found the issue thanks to your help.

Inside .wslconfig this is the only line I had:
```
[wsl2]
guiApplications=false
```
and this was preventing wslg to start. I have no idea why it's set to false. I set it to true and restarted wsl and it works now.
```wsl --system``` returned this error and this is the detail that gave it away

```GUI application support is disabled via C:\Users\blabla\.wslconfig or /etc/wsl.conf.
Error code: Wsl/Service/CreateInstance/WSL_E_GUI_APPLICATIONS_DISABLED```
